docs/install.md

```diff
@@ -11,23 +11,29 @@ for example:
 ```
 $ gsutil ls -r "gs://terraform-validator/releases/v*"
 ...
-gs://terraform-validator/releases/v0.12.5/:
-gs://terraform-validator/releases/v0.12.5/terraform-validator_darwin_amd64-0.12.5.tar.gz
-gs://terraform-validator/releases/v0.12.5/terraform-validator_darwin_arm64-0.12.5.tar.gz
-gs://terraform-validator/releases/v0.12.5/terraform-validator_linux_amd64-0.12.5.tar.gz
-gs://terraform-validator/releases/v0.12.5/terraform-validator_linux_arm64-0.12.5.tar.gz
-gs://terraform-validator/releases/v0.12.5/terraform-validator_windows_amd64-0.12.5.tar.gz
+gs://terraform-validator/releases/v0.13.0/:
+gs://terraform-validator/releases/v0.13.0/terraform-validator_darwin_amd64-0.13.0.tar.gz
+gs://terraform-validator/releases/v0.13.0/terraform-validator_darwin_arm64-0.13.0.tar.gz
+gs://terraform-validator/releases/v0.13.0/terraform-validator_linux_amd64-0.13.0.tar.gz
+gs://terraform-validator/releases/v0.13.0/terraform-validator_linux_arm64-0.13.0.tar.gz
+gs://terraform-validator/releases/v0.13.0/terraform-validator_windows_amd64-0.13.0.tar.gz
 ```
 
 To download the binary, you need to
 [install](https://cloud.google.com/storage/docs/gsutil_install#install) the
 `gsutil` tool first. The following commands download and uncompress the Linux AMD64
-version of Terraform Validator from v0.12.5 release to your local directory:
+version of Terraform Validator from v0.13.0 release to your local directory:
 
 ```
-gsutil cp gs://terraform-validator/releases/v0.12.5/terraform-validator_linux_amd64-0.12.5.tar.gz .
-tar -xzvf terraform-validator_linux_amd64-0.12.5.tar.gz
+gsutil cp gs://terraform-validator/releases/v0.13.0/terraform-validator_linux_amd64-0.13.0.tar.gz .
+tar -xzvf terraform-validator_linux_amd64-0.13.0.tar.gz
 chmod 755 terraform-validator
 ```
 
 The full list of releases, with release notes, is available [on Github](https://github.com/GoogleCloudPlatform/terraform-validator/releases).
+
+Binary builds are only available for versions up to 0.13.0. 0.14.0+ are only available via [`gcloud beta terraform vet`](https://cloud.google.com/docs/terraform/policy_validation)
+
+## Disclaimer
+
+This is not an officially supported Google product.
```

Comment:
Perhaps put this at the top of the page within a banner to warn the user. Then this would be the first thing they see within the page.

Reply:
I think having this at the bottom of the page might be a best practice, but I can put it at the top as well; that seems reasonable.